testrepository/tests/repository/test_file.py

```diff
@@ -19,6 +19,7 @@
 import tempfile
 
 from fixtures import Fixture
+import mock
```

Comment:
I don't believe mock is a documented dependency today, and so this should be added and documented. Or, perhaps use a fixtures patch instead.